tsl/test/shared/expected/dist_chunk.out

```diff
@@ -3,30 +3,38 @@
 -- LICENSE-TIMESCALE for a copy of the license.
 -- This file contains tests for all features that will be used as part
 -- of the chunk move/copy multi-node functionality
--- Test function _timescaledb_internal.create_chunk_replica_table
 -- A table for the first chunk will be created on the data node, where it is not present.
 SELECT chunk_name, data_nodes 
 FROM timescaledb_information.chunks 
-WHERE hypertable_name = 'dist_chunk_copy';
+WHERE hypertable_name = 'dist_chunk_copy'
+ORDER BY 1, 2;
        chunk_name        |        data_nodes         
 -------------------------+---------------------------
- _dist_hyper_X_X_chunk | {data_node_1,data_node_2}
  _dist_hyper_X_X_chunk | {data_node_2,data_node_3}
+ _dist_hyper_X_X_chunk | {data_node_2,data_node_3}
+ _dist_hyper_X_X_chunk | {data_node_1,data_node_3}
+ _dist_hyper_X_X_chunk | {data_node_1,data_node_3}
+ _dist_hyper_X_X_chunk | {data_node_2,data_node_3}
+ _dist_hyper_X_X_chunk | {data_node_1,data_node_2}
  _dist_hyper_X_X_chunk | {data_node_1,data_node_3}
+ _dist_hyper_X_X_chunk | {data_node_2,data_node_3}
  _dist_hyper_X_X_chunk | {data_node_1,data_node_2}
  _dist_hyper_X_X_chunk | {data_node_2,data_node_3}
-(5 rows)
+ _dist_hyper_X_X_chunk | {data_node_1,data_node_3}
+(11 rows)
 
 -- Non-distributed chunk will be used to test an error
 SELECT chunk_name 
 FROM timescaledb_information.chunks 
-WHERE hypertable_name = 'conditions';
+WHERE hypertable_name = 'conditions'
```

Comment:
I guess we don't need an order by with that output ;-) but good for consistency I guess

Reply:
I'll add just to be sure